### PR TITLE
Made first part of the home work.

### PR DESCRIPTION
**Pull request by olgaz-skiba**

### CODE REVIEW

File: homework/english/skiba_olga/lecture1/task1.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#Homework 1 task 1
```
Comment (olgaz-skiba) [inline homework/english/skiba_olga/lecture1/task1.md:1]: This is not a homework. Please do homework!

- [ ] add at list 1 task

**Review by olgaz-skiba — COMMENTED** [review]
See notes

**Review by Denis-Pikus — APPROVED** [review]
lgfm

**Review by Alya-Homyak — APPROVED** [review]

**Review by Alexandr-R-94 — APPROVED** [review]

**Comment by olgaz-skiba** [comment]
Please, Can you tell me why you closed my request. Thanks. And what does it mean

**Comment by LiudmilaZhdanok** [comment]
Hi!
When I open your task for review it has been merged yet. So I didn't check
it again.
May be I don't understand something with GitHub, then I'm sorry

вт, 19 янв. 2021 г., 02:27 olgaz-skiba <notifications@github.com>:

File: homework/english/skiba_olga/lecture1/task1.md (hunk 1 above)
Comment (andrpech) [reply homework/english/skiba_olga/lecture1/task1.md:1]: lgfm

**Review by andrpech — APPROVED** [review]

**Pull request merged**